docs/development.md

```diff
@@ -108,12 +108,12 @@ in your boxes.yaml entry to configure this on box creation.
 
 #### Custom files from git repo
 
-A git repo's contents can be copied to the target user's home directory when spinning up a `centos8-katello-devel` or `centos8-katello-devel-stable` box. This can be done by specifying the `customize_home_git_repo` ansible variable. For example:
+A git repo's contents can be copied to the target user's home directory when spinning up a `centos9-katello-devel` or `almalinux8-katello-devel-stable` box. This can be done by specifying the `customize_home_git_repo` ansible variable. For example:
```

Comment:
Please consistently point to centos9

Reply:
@ekohl i've just re-reviewed all the changes, hopefully this time i haven't missed anything this time :/